### ISSUE TITLE
Cannot read property 'webContents' of null

### ISSUE THREAD

**Issue by page1** [AugurProject/augur-app#271]
On Clicking "Connect"

```
Uncaught Exception:
TypeError: Cannot read property 'webContents' of null
    at AugurUIServer.startServer (/Applications/Augur.app/Contents/Resources/app/src/main/augurUIServer.js:72:17)
    at AugurUIServer.onStartUiServer (/Applications/Augur.app/Contents/Resources/app/src/main/augurUIServer.js:27:34)
    at emitTwo (events.js:126:13)
    at EventEmitter.emit (events.js:214:7)
    at WebContents.<anonymous> (/Applications/Augur.app/Contents/Resources/electron.asar/browser/api/web-contents.js:286:13)
    at emitTwo (events.js:126:13)
    at WebContents.emit (events.js:214:7)
```

<img width="953" alt="screen shot 2018-07-31 at 10 00 46 am" src="https://user-images.githubusercontent.com/6609556/43471630-ab305c28-94a8-11e8-83a3-e6e13808ca23.png">

**Comment by sharkcrayon** [comment]
@page1 Which version of the app are you on?

**Comment by sharkcrayon** [comment]
https://app.clubhouse.io/augur/story/14831/github-cannot-read-property-webcontents-of-null

**Comment by page1** [comment]
@sharkcrayon 1.0.7

**Comment by bthaile** [comment]
This will be fixed in the next version, to be coming out soon.

**Comment by bthaile** [comment]
addressed in 1.4
